json-endpoints/class.wpcom-json-api-list-posts-v1-1-endpoint.php

```diff
@@ -68,7 +68,7 @@ function callback( $path = '', $blog_id = 0 ) {
 		}
 
 		if ( isset( $args['type'] ) &&
-			   ! in_array( $args['type'], array( 'post', 'page', 'revision', 'any' ) ) &&
+			   ! in_array( $args['type'], array( 'post', 'revision', 'page', 'any' ) ) &&
```

Comment:
this appears to be a bit different then my original wpcom PR, that merge involved removing page from these arrays, so this line was:

`! in_array( $args['type'], array( 'post', 'revision', 'any' ) ) &&`

See https://github.com/Automattic/jetpack/pull/7006/commits/5b6081f75940b1b713283863d7ddcdbb2dfca1c8